java/BinExport/src/main/java/com/google/security/binexport/BinExport2Builder.java

```diff
@@ -311,8 +311,15 @@ private void buildFlowGraphs(Map<Long, Integer> basicBlockIndices)
               edge.setTargetBasicBlockIndex(targetId);
             }
             edges.add(edge.build());
+            // Possibly naive (and experimental) handling for switch tables
```

Comment:
Please remove this comment. That is better placed in the commit message (which already says "preliminary").